resume-buddy/src/test/java/com/google/sps/MatchTest.java

```diff
@@ -235,4 +235,30 @@ public void isNotMatchAndStillHaveTime() throws ParseException {
 
     return actualMatches;
   }
+
+  /** Test sending emails when they are matched */
+  @Test
+  public void testSendingEmailWhenMatched() throws ParseException {
+    // Uncomment when needed to test. Otherwise, will send too much emails.
+    // sendingEmailsWhenMatched();
+  }
+
+  private void sendingEmailsWhenMatched() {
+    REVIEWEE_A.setProperty("email", "animachaidze@gmail.com");
```

Comment:
If you want to use test gmail accounts to receive and send emails, I can make test @gmail.com accounts, so you don't spam your personal emails.